jwst/__init__.py

```diff
@@ -10,3 +12,10 @@
     commit = _regex_git_hash.match(__version__).groups()
     if commit:
         __version_commit__ = commit[0]
+
+if "CRDS_CONTEXT" not in os.environ:
```

Comment:
A comment will be added here describing this structure if we go with this approach.